columnphysics/icepack_therm_vertical.F90

```diff
@@ -2612,6 +2612,13 @@ subroutine icepack_step_therm1(dt,                        &
       massicen(:,:) = c0
       massliqn(:,:) = c0
 
+      if(geos_heatflux) then
+         fswthru_uvrdr = c0
+         fswthru_uvrdf = c0
+         fswthru_pardr = c0
+         fswthru_pardf = c0
+      endif
+
```

Comment:
Why is this needed for the uvr and par sw but not for the visible and infrared terms?

Reply:
They have to be set to zero somewhere prior to being  passed to ```merge_flux```. This resetting was done for vis and nir counterparts in ```init_flux_ocn``` which gets called every time step. Maybe this should also move to  ```init_flux_ocn```?

Reply:
Moving to init_flux_ocn would be great if it works.  My preference is to follow the same implementation strategy as the existing visible and infrared terms if it's possible.

Reply:
Will do!

Reply:
There is an extra blank line here, please remove.  Also, don't see the changes in alloc_flux.  Maybe still TBD?